control-plane-operator/controllers/hostedcontrolplane/kas/config.go

```diff
@@ -99,7 +99,7 @@ func generateConfig(p KubeAPIServerConfigParams) *kcpv1.KubeAPIServerConfig {
 									Kind:       "PodSecurityConfiguration",
 								},
 								Defaults: podsecurityadmissionv1beta1.PodSecurityDefaults{
-									Enforce:        "restricted",
+									Enforce:        "privileged",
```

Comment:
We need to bump `github.com/openshift/api` go module to pick up https://github.com/openshift/api/pull/2120/ in order to align with what we did for https://github.com/openshift/hypershift/pull/4834.